src/main/java/org/nguh/nguhcraft/block/NguhBlocks.kt

```diff
@@ -235,14 +227,190 @@ object NguhBlocks {
+
+    val CHISELED_CALCITE_BRICKS = Register(
+        "chiseled_calcite_bricks",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
+
+    // =========================================================================
+    // Gilded calcite
+    // =========================================================================
+    val GILDED_CALCITE = Register(
+        "gilded_calcite",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
+
+    val GILDED_CALCITE_SLAB = RegisterVariant(GILDED_CALCITE, "slab", ::SlabBlock)
+    val GILDED_CALCITE_STAIRS = RegisterStairs(GILDED_CALCITE)
+
+    val GILDED_POLISHED_CALCITE = Register(
+        "gilded_polished_calcite",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
+
+    val GILDED_POLISHED_CALCITE_SLAB = RegisterVariant(GILDED_POLISHED_CALCITE, "slab", ::SlabBlock)
+    val GILDED_POLISHED_CALCITE_STAIRS = RegisterStairs(GILDED_POLISHED_CALCITE)
+    val GILDED_POLISHED_CALCITE_WALL = RegisterVariant(GILDED_POLISHED_CALCITE, "wall", ::WallBlock)
+
+    val GILDED_CHISELED_CALCITE = Register(
+        "gilded_chiseled_calcite",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
+
+    val GILDED_CALCITE_BRICKS = Register(
+        "gilded_calcite_bricks",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
+
+    val GILDED_CALCITE_BRICK_SLAB = RegisterVariant(GILDED_CALCITE_BRICKS, "slab", ::SlabBlock)
+    val GILDED_CALCITE_BRICK_STAIRS = RegisterStairs(GILDED_CALCITE_BRICKS)
+    val GILDED_CALCITE_BRICK_WALL = RegisterVariant(GILDED_CALCITE_BRICKS, "wall", ::WallBlock)
+
+    val GILDED_CHISELED_CALCITE_BRICKS = Register(
+        "gilded_chiseled_calcite_bricks",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
+
+    // =========================================================================
     // Block entities.
+    // =========================================================================
     val LOCKED_DOOR_BLOCK_ENTITY = RegisterEntity(
         "lockable_door",
         FabricBlockEntityTypeBuilder
             .create(::LockedDoorBlockEntity, LOCKED_DOOR)
             .build()
     )
 
+    // =========================================================================
+    // Block families
+    // =========================================================================
+    val CINNABAR_FAMILY = BlockFamilies.register(NguhBlocks.CINNABAR)
+        .polished(NguhBlocks.POLISHED_CINNABAR)
+        .slab(NguhBlocks.CINNABAR_SLAB)
+        .stairs(NguhBlocks.CINNABAR_STAIRS)
```

Comment:
```suggestion
        .polished(NguhBlocks.POLISHED_CINNABAR)
        .slab(NguhBlocks.CINNABAR_SLAB)
        .stairs(NguhBlocks.CINNABAR_STAIRS)
```
When I said ‘remove `NguhBlocks.` everywhere, I meant everywhere in this file.